helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -847,7 +847,12 @@ exporters:
 service:
   telemetry:
     metrics:
-      address: 0.0.0.0:8889
+      readers:
```

Comment:
Is this new config interface also default for the version we are on? I think 8889 is our custom port so we probably have to add this anyway but just curious

Reply:
Yeah, that's right. Default is the same, just different 8888 port